README.md

```diff
@@ -92,6 +92,7 @@ The solutions are located under `/LeetcodeProblems`. Each problem has a test fil
 | [Award Budget Cuts](/LeetcodeProblems/Algorithms/Award_Budget_Cuts.js)                                          |        | |
 | [Happy Number](https://leetcode.com/problems/happy-number/)                                                     | Easy   | https://leetcode.com/problems/happy-number/ |
 | [Shuffle String](https://leetcode.com/problems/shuffle-string/)                                                 | Easy   | https://leetcode.com/problems/shuffle-string/ |
+| [K Closest Points to Origin](/LeetcodeProblems/Algorithms/K_Closest_Points_to_Origin.js/)                       | Medium | https://leetcode.com/problems/k-closest-points-to-origin/
```

Comment:
nit: This list is sorted by hard/medium/easy, could you add it to the Medium section?

Reply:
Yes yes I can do that!